modules/registry-configuring-storage-baremetal.adoc

```diff
@@ -24,7 +25,7 @@ the `configs.imageregistry/cluster` resource.
 . Verify you do not have a registry pod:
 +
 ----
-$ oc get pods -n configs.imageregistry.operator.openshift.io/cluster 
+$ oc get pods -n configs.imageregistry.operator.openshift.io/cluster
```

Comment:
Obviously the above command is using wrong syntax. 
Here you want to verify if a image registry config - 'cluster' is existing for the following modification, but not pod. am I right?
If you want to verify registry existing or not, should be:
`$ oc get po -n openshift-image-registry`